### ISSUE TITLE
Provide imports for testing and large codebase migrations

### ISSUE THREAD

**Issue by fdietze** [OutWatch/outwatch#76]
Currently it is not possible to manually write data into a `Sink` or `Handler`. For safety reasons the `next()` method on the inner observable of Sink is hidden. This is a good thing to discourage the usage of side-effects.

Though this would be useful to use in tests.

Another use-case is migration of large code-bases. Allowing side-effecting parts of the code to write into `Sink`/`Handler` would help to at least compile a partially migrated code base.

I suggest an Import like
```scala
import outwatch.sinkUnsafeNext
```

where `SinkUnsafeNext` is an implicit class over Sink to provide an `unsafeNext()` method.

**Comment by mariusmuja** [comment]
You could construct and ObserverSink that allows you to push items into it (has the inner observer public):
```scala
  def unsafeSink[T](sink: Sink[T]): IO[ObserverSink[T]] = {
    val subject = Subject[T]
    val newSink = ObserverSink(subject)

    (sink <-- subject).map(_ => newSink)
  }
```

For example:

```scala
def main(args: Array[String]): Unit = {

    val handler = Handlers.createStringHandler()

    handler.flatMap { handler =>
      handler(elem => dom.console.log(s"Elem: $elem"))

      unsafeSink(handler).map { sink =>
        sink.observer.next("One")
        sink.observer.next("Two")
        sink.observer.next("Three")
      }
    }.unsafeRunSync()
  }
```

**Comment by cornerman** [comment]
But why is the `observer` in `ObserverSink` public, but not in `Sink`?

**Comment by cornerman** [comment]
Fixed with #149
